fprodents/patch/workflows/anatomical.py

```diff
@@ -423,16 +423,16 @@ def _check_img(img):
     mrg_tpms = pe.Node(niu.Merge(3), name="mrg_tpms")
 
     anat_dseg = pe.Node(
-        fsl.FAST(segments=True, no_bias=True, probability_maps=True),
+        FAST(
+            segments=True,
+            probability_maps=True,
+            bias_iters=0,
+            no_bias=True,
+        ),
         name="anat_dseg",
         mem_gb=3,
     )
-    # Change LookUp Table - BIDS wants: 0 (bg), 1 (gm), 2 (wm), 3 (csf)
-    lut_anat_dseg = pe.Node(
-        niu.Function(function=_apply_bids_lut), name="lut_anat_dseg"
-    )
 
-    lut_anat_dseg.inputs.lut = (0, 3, 2, 1)  # Maps: 0 -> 0, 3 -> 1, 2 -> 2, 1 -> 3.
```

Comment:
If there output labels are already what we expect (comment in line 430), then let's make this node do nothing
```suggestion
    lut_anat_dseg.inputs.lut = (0, 1, 2, 3)  # Maps: 0 -> 0, 3 -> 1, 2 -> 2, 1 -> 3.
```

Reply:
I think this was actually based on some changes that I was testing (using FAST with T2 as the image type),so I've reverted it to it's original incarnation here